docs-source/content/userguide/managing-domains/model-in-image/overview.md

```diff
@@ -34,44 +35,34 @@ It enables:
 
 This feature is supported for standard WLS domains, Restricted JRF domains, and JRF domains.
 
-WDT models are a convenient and simple alternative to WebLogic WLST configuration scripts and templates. They compactly define a WebLogic domain using YAML files and support including application archives in a ZIP file.  The WDT model format is described in the open source, [WebLogic Deploy Tool](https://github.com/oracle/weblogic-deploy-tooling) GitHub project.
+For JRF domains, Model in Image provides additional support for initializing the infrastructure database for a domain, when a domain is started for the first time, supplying an database password, and obtaining an database wallet for re-use in subsequent restarts of the same domain. See [Requirements for JRF domain types]({{< relref "/userguide/managing-domains/model-in-image/usage/_index.md#requirements-for-jrf-domain-types" >}}).
```

Comment:
Question here: (I'm using semi-colons only to highlight the difference in phrases), is it "additional support for initializing the infrastructure database for a domain when a domain is started for the first time; supplying a database password; and obtaining an database wallet" OR is it: "additional support for initializing the infrastructure database for a domain; when a domain is started for the first time, supplying a database password; and obtaining an database wallet"? IF the correct phrasing is the latter, then remove the comma after domain, before when.